### DISCUSSION TITLE
Possible bug?

### DISCUSSION THREAD

**Discussion by jdfoote** [Extended-Thunder/send-later#721] [General]
After updating to TB 128.0, Send Later appears to not be working.

I have an email which passed its send date and wasn't sent.

Also, when I click on the pane, nothing appears, even though it looks like it is tracking some emails.

![image](https://github.com/user-attachments/assets/e69284a8-330b-4468-b0a6-7adce27f5aa9)

**Comment by jikamens** [comment]
Make sure you have Send Later 10.5.1.

If you do and this is still happening. please follow the [instructions in the user guide](https://extended-thunder.github.io/send-later/#console-logging) for capturing console logging at the All level. At step 5 in those instructions, schedule a new message with Send Later and then click the button in the main window to see if it shows up there. When you're done, send the log file to send-later-support@extended-thunder.org. If it's over 1MB then please upload it to Google Drive or something and then share the file with me from there rather than attaching it to an email message. Thanks.

**Reply by jdfoote** [reply]
Oh boy. I realized the email wasn't scheduled until next month! :)

Sending is working, it's just the pane that isn't working. I believe this is the relevant section from the logs. LMK if you'd like me to email it as well.

```
sendRemoveListener on closed conduit sendlater3@kamens.us.274877913802 [ConduitsChild.sys.mjs:122:13](resource://gre/modules/ConduitsChild.sys.mjs)
    _send resource://gre/modules/ConduitsChild.sys.mjs:122
    removeListener resource://gre/modules/ExtensionChild.sys.mjs:673
    removeListener resource://gre/modules/ExtensionChild.sys.mjs:929
    register chrome://extensions/content/child/ext-storage.js:163
    removeListener resource://gre/modules/ExtensionCommon.sys.mjs:2957
    revoke resource://gre/modules/ExtensionCommon.sys.mjs:2979
    close resource://gre/modules/ExtensionCommon.sys.mjs:2984
    unload resource://gre/modules/ExtensionCommon.sys.mjs:1019
    unload resource://gre/modules/ExtensionPageChild.sys.mjs:281
    unload resource://gre/modules/ExtensionPageChild.sys.mjs:324
    destroyExtensionContext resource://gre/modules/ExtensionPageChild.sys.mjs:496
    observe resource://gre/modules/ExtensionPageChild.sys.mjs:397
```

Thanks so much for your work on this!

**Comment by jikamens** [comment]
I don't think that's the problem.
Please follow the instructions above for capturing logs and I'll take a look.

**Comment by castagnag** [comment]
I confirm this bug. I have the same problem on three different computers (two Thinkpads and a Precision Dell). They all run Fedora 40, with Wayland, TB 128.2.0esr, and Send Later 10.6.0. Here is my screenshot.
![Screenshot from 2024-10-02 13-48-15](https://github.com/user-attachments/assets/4a320c7d-e50e-475b-8e04-e51c1e116009)

**Comment by jikamens** [comment]
There are two different problems described here -- the blank pop-up and the messages not sending -- and I don't know which one you're referring to when you say you "confirm _this_ bug" and "have the _same_ problem". Which problem are you referring to?

If you're referring to scheduled messages not sending, then as noted above, I need logs to be able to troubleshoot further.

If you are referring to the blank pop-up, then that's [addressed in the release notes](https://extended-thunder.github.io/send-later/release-notes.html#issue-wayland).

**Comment by jdfoote** [comment]
Thanks so much - changing  `widget.wayland.use-move-to-rect` to `false` in `about:config` worked for me!

**Comment by castagnag** [comment]
Hi, 
    I was referring to the blank pop-up, and, yes changing widget.wayland.use-move-to-rect solved the issue. Thanks a lot!
